.github/workflows/main.yml

```diff
@@ -70,13 +70,10 @@ jobs:
           python --version
           cmake --version
 
-      # 2024.10.15: This step can be adjusted or even deleted if we decide to use a higher G++ or protobuf version.
-      # With the current protobuf version, g++13 does not compile due to missing headers
       - name: Show versions g++
         if: startsWith(matrix.os,'ubuntu')
         run: |
-          sudo apt install g++-11
-          sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-11 12
+          sudo apt install g++
```

Comment:
Do we still need to install, or can we use the stock version ?

Reply:
I think we can use the stock version